UndertaleModLib/Decompiler/Instructions/Decompiler.ExpressionTwo.cs

```diff
@@ -34,15 +34,69 @@ public override Statement CleanStatement(DecompileContext context, BlockHLStatem
             return this;
         }
 
-        public override string ToString(DecompileContext context)
+        private string AddParensIfNeeded(Expression argument, DecompileContext context)
+        {
+            string arg = argument.ToString(context);
+            bool needsParens;
+            if (arg.Substring(0, 1) != "(" &&
+                argument is not ExpressionConstant &&
+                arg.Contains(' ', StringComparison.InvariantCulture))
+                needsParens = true;
+            else
+                needsParens = false;
+
+
+            if (argument is ExpressionTwo argumentAsBinaryExpression)
+            {
+                int myPriorityLevel = Opcode switch
```

Comment:
"myprioritylevel" is a bit of a weird variable name. These sound more like they should be "firstArgumentPriorityLevel" and "secondPriorityLevel"?